SPECS/openblas/openblas.spec

```
@@ -422,17 +384,16 @@ make -C openmp64_   $TARGET USE_THREAD=1 USE_OPENMP=1 FC=gfortran CC=gcc COMMON_
 %endif
 
 %install
-rm -rf %{buildroot}
 # Install serial library and headers
-make -C serial USE_THREAD=0 PREFIX=%{buildroot} OPENBLAS_LIBRARY_DIR=%{buildroot}%{_libdir} OPENBLAS_INCLUDE_DIR=%{buildroot}%{_includedir}/%name OPENBLAS_BINARY_DIR=%{buildroot}%{_bindir} OPENBLAS_CMAKE_DIR=%{buildroot}%{_libdir}/cmake install
+make -C serial USE_THREAD=0 PREFIX=%{buildroot} OPENBLAS_LIBRARY_DIR=%{buildroot}%{_libdir} OPENBLAS_INCLUDE_DIR=%{buildroot}%{_includedir}/%{name} OPENBLAS_BINARY_DIR=%{buildroot}%{_bindir} OPENBLAS_CMAKE_DIR=%{buildroot}%{_libdir}/cmake install
 
 # Copy lapacke include files
 %if %{with system_lapack} && %{lapacke}
 cp -a %{_includedir}/lapacke %{buildroot}%{_includedir}/%{name}
 %endif
 
 # Fix i686-x86_64 multilib difference
-%multilib_fix_c_header --file %{_includedir}/openblas/openblas_config.h
+%{multilib_fix_c_header} --file %{_includedir}/openblas/openblas_config.h
```

This line change should be reverted. It's one of the remaining issues with out linter: it doesn't differentiate between macros, that are just simple values/strings, and ones that are functions accepting arguments. Surrounding the latter in curly brackets makes them being run with no arguments and thing end up breaking or misconfigured. Having said that, I don't see this macro even being defined anywhere, so it might be worth taking a look at the build logs, but for this review I'd just need to remove the brackets.

```suggestion
%multilib_fix_c_header --file %{_includedir}/openblas/openblas_config.h
```